src/e2e-test/regression-test/util/utils.js

```diff
@@ -0,0 +1,78 @@
+// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+// TODO: consider using `util` nodule inside node env
+const _ = require('lodash');
+const handlers = require('./handlers');
+const MAX_COMPARISION_DEPTH = 10;
```

Comment:
The threshold is to prevent unlimited recursing. After reading some ground truth files, I found the object depth will not exceed 6, so I use 10 as the threshold. Should I change a bigger number ?

Reply:
Yes, we can change to 100.